doc/classes/AnimationMixer.xml

```diff
@@ -36,6 +36,18 @@
 				Manually advance the animations by the specified time (in seconds).
 			</description>
 		</method>
+		<method name="capture">
+			<return type="void" />
+			<param index="0" name="name" type="StringName" />
+			<param index="1" name="duration" type="float" />
+			<param index="2" name="trans_type" type="int" enum="Tween.TransitionType" default="0" />
+			<param index="3" name="ease_type" type="int" enum="Tween.EaseType" default="0" />
+			<description>
+				If the animation track specified by [param name] has an option [constant Animation.UPDATE_CAPTURE], stores current values of the objects indicated by the track path as a cache. If there is already a captured cache, the old cache is discarded.
+				After, it will interpolate with current animation blending result during the playback process for the time specified by [param duration], working like a crossfade.
```

Comment:
```suggestion
				After this it will interpolate with current animation blending result during the playback process for the time specified by [param duration], working like a crossfade.
```